dynatrace-oneagent-operator/templates/Common/customresource-oneagent.yaml

```diff
@@ -58,8 +58,26 @@ spec:
   {{- end }}
 
   {{- if .Values.oneagent.resources }}
-  resources: {{- toYaml .Values.oneagent.resources | nindent 4 }}
-  {{- end }}
+  resources:
+    {{ if or (.Values.oneagent.resources.requests.cpu) (.Values.oneagent.resources.requests.memory) }}
```

Comment:
Same question about the dashes.

Reply:
Did work without the dashes but added them just in case